internal/artifact/follow/follow.go

```diff
@@ -261,3 +270,33 @@ func (o *artifactFollowOptions) retrieveFalcoVersions(ctx context.Context) error
 
 	return nil
 }
+
+type backoffTransport struct {
+	Base     http.RoundTripper
+	Printer  *output.Printer
+	MaxRetry uint
+	attempts uint
+	sleep    time.Duration
+}
+
+func (v *backoffTransport) RoundTrip(req *http.Request) (*http.Response, error) {
+	var err error
+	var resp *http.Response
+
+	v.sleep = 1 * time.Second
+
+	for v.attempts < v.MaxRetry {
+		v.attempts++
+		v.Printer.Verbosef("retrieving Falco versions, attempt %d\n", v.attempts)
+		resp, err = v.Base.RoundTrip(req)
+		if err != nil {
+			v.Printer.Verbosef("error: %s. Trying again in %s", err.Error(), v.sleep.String())
+			time.Sleep(v.sleep)
```

Comment:
What i don't like about this solutions is that we end up sleeping `2<<5` before actually breaking and returning the error.